plugins/modules/pnp_intent.py

```diff
@@ -865,7 +908,7 @@ class instance for further use.
                     "payload": update_payload},
             op_modifies=True,
         )
-        self.log(str(update_response))
+        self.log("Update in the device's config API response is {0}".format(str(update_response)), "DEBUG")
```

Comment:
self.log("Response from 'update_device' API for device's config update: {0}".format(str(update_response)), "DEBUG")

Reply:
Resolved